app/src/main/java/org/hackillinois/android/database/Database.kt

```diff
@@ -11,7 +11,8 @@ import org.hackillinois.android.database.entity.*
         Attendee::class,
         User::class,
         Event::class,
-        Roles::class
+        Roles::class,
+        ProjectModel::class
```

Comment:
Maybe this should be called Project and not ProjectModel? (for consistency with the other classes)

Reply:
Agreed. Can you make this change @RaghavSaini01 ?

Reply:
Yeah sure, I just realized I changed the file name but not the class name. Will fix that